karmada-Failover-example/step2/text.md

```diff
@@ -0,0 +1,13 @@
+### create deployment
+
+create deployment named nginx
+
+   RUN `kubectl --kubeconfig /etc/karmada/karmada-apiserver.config apply -f ~/nginx/nginxDeployment.yaml`{{exec}}
```

Comment:
Please add the `cat` command to display the details of the YAML file.

Reply:
fixed

Reply:
also `cat` the PropagationPolicy file.